scene/realistic-lighting-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsController.java

```diff
@@ -103,10 +103,8 @@ public void initialize() {
       comboBox.setConverter(new ComboBoxStringConverter());
       comboBox.setCellFactory(comboBox -> new LightingModeListCell());
 
-      // update the sun lighting based on the lighting mode chosen from the combo box
-      comboBox.getSelectionModel().selectedItemProperty().addListener(e -> {
-        sceneView.setSunLighting(comboBox.getSelectionModel().getSelectedItem());
-      });
+      // bind the sun lighting to the lighting mode chosen from the combo box
+      comboBox.valueProperty().bindBidirectional(sceneView.sunLightingProperty());
```

Comment:
This is fine. For the other property, I have an idea you can try. I'll send you some thoughts. I'd also see if you can put this sample into a separate PR so it's easier for someone else to review rather than needing to go through 4 samples, as there may be some more experience we can gain here.

Reply:
Also I think we should switch this to start on "lighting and shadows" as I always go straight to the slider and it does nothing.